### PR TITLE
Node: added PFCOUNT command

### PR DESCRIPTION
**Pull request by yipin-chen**
https://redis.io/docs/latest/commands/pfcount/

### CODE REVIEW

File: node/src/BaseClient.ts [hunk 1]
```diff
@@ -2380,6 +2381,24 @@ export class BaseClient {
         return this.createWritePromise(createPfAdd(key, elements));
     }
 
+    /** Estimates the cardinality of the data stored in a HyperLogLog structure for a single key or
+     * calculates the combined cardinality of multiple keys by merging their HyperLogLogs temporarily.
+     *
+     * See https://valkey.io/commands/pfcount/ for more details.
+     *
```
Comment (Yury-Fridlyand) [inline node/src/BaseClient.ts:2388]: Add a cross slot notice

**Review by Yury-Fridlyand — APPROVED** [review]

File: node/src/BaseClient.ts [hunk 2]
```diff
@@ -2380,6 +2381,25 @@ export class BaseClient {
...
+     * @remarks When in cluster mode, all `keys` must map to the same hash slot.
+     * @param keys - The keys of the HyperLogLog data structures to be analyzed.
+     * @returns - The approximated cardinality of given HyperLogLog data structures.
+     *     The cardinality of a key that does not exist is 0.
+     * @example
+     * ```typescript
+     * const result = await client.pfcount("hll_1", "hll_2");
```
Comment (acarbonetto) [inline node/src/BaseClient.ts:2395]:
```suggestion
     * const result = await client.pfcount(["hll_1", "hll_2"]);
```

File: node/tests/RedisClusterClient.test.ts [hunk 3]
```diff
@@ -288,6 +288,7 @@ describe("RedisClusterClient", () => {
                 client.smove("abc", "zxy", "value"),
                 client.renamenx("abc", "zxy"),
                 client.sinter(["abc", "zxy", "lkn"]),
+                client.pfcount(["def", "ghi"]),
```
Comment (acarbonetto) [inline node/tests/RedisClusterClient.test.ts:291]:
```suggestion
                client.pfcount(["abc", "zxy", "lkn"]),
```

**Review by acarbonetto — APPROVED** [review]
please fix the examples

**Pull request merged**